index.html

```diff
@@ -11,70 +11,180 @@
+                    </ul>
+                
+                    <button type="button" class="see-project">See Project</button>
+                </div>
+            </section>
+            <section>
+                <img src="./image/Img-Placeholder.svg" alt="image placeholder" class="image-placeholder">
+                <div>
+                    <h1 class="multi-post-header">Multi-Post Stories</h1>
+                    <p class="multi-post-paragraph">A daily section of privately personalized reads; no accounts
+                        or sign-ups required. has been the industry&#x27;s standard
+                        dummy text ever since the 1500s, when an unknown printer took a 
+                        standard dummy text.
+                    </p>
+
+                    <ul class="icon">
+                        <li><img src="./image/css.svg" alt="css-icon"></li>
+                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                        <li><img src="./image/html.svg" alt="html-icon"></li>
+                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                        <li><img src="./image/bootstrap.svg" alt="bootstrap-icon"></li>
+                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                        <li><img src="./image/Ruby.svg" alt="ruby-icon"></li>
+                    </ul>
+                
+                    <button type="button" class="see-project">See Project</button>
+                </div>
+            </section>
+            <section>
+                <img src="./image/Img-Placeholder.svg" alt="image placeholder" class="image-placeholder">
+                <div>
+                    <h1 class="multi-post-header">Multi-Post Stories</h1>
+                    <p class="multi-post-paragraph">A daily section of privately personalized reads; no accounts
+                        or sign-ups required. has been the industry&#x27;s standard
+                        dummy text ever since the 1500s, when an unknown printer took a 
+                        standard dummy text.
+                    </p>
+
+                    <ul class="icon">
+                        <li><img src="./image/css.svg" alt="css-icon"></li>
+                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                        <li><img src="./image/html.svg" alt="html-icon"></li>
+                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                        <li><img src="./image/bootstrap.svg" alt="bootstrap-icon"></li>
+                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                        <li><img src="./image/Ruby.svg" alt="ruby-icon"></li>
+                    </ul>
+                
+                    <button type="button" class="see-project">See Project</button>
+                </div>
+            </section>
         </div>
```

Comment:
- [ ] Kindly be noticed that the works container of the section is a `section` tag.